addons/better-quoter/userscript.js

```diff
@@ -4,13 +4,19 @@ export default async function ({ addon, global, console }) {
     let quoteButton = await addon.tab.waitForElement(".postquote a", { markAsSeen: true });
     quoteButton.setAttribute("onclick", "return false");
     quoteButton.addEventListener("mouseup", (e) => {
-      let blockpost = e.path.find((e) => e.classList.contains("blockpost"));
-      if (window.getSelection().toString().length)
-        textarea.value += `[quote=${
-          blockpost.querySelector(".black.username").innerText
-        }]${window.getSelection().toString()}[/quote]`;
+      let blockpost = quoteButton.closest(".blockpost");
```

Comment:
Can you add a `.querySelector(".post_body_html")` (should work) on the end of here so edit timestamps , siggies and user info can't be quoted? Thanks.